lib/Table/TableColumn.ts

```diff
@@ -592,7 +603,16 @@ export default class TableColumn {
     return (
       this.tableModel.columnTitles[this.columnNumber] ??
       this.traits.title ??
+      // If no title set, use `name` and:
+      // - un-camel case
+      // - remove underscores
+      // - capitalise
       this.name
+        .replace(/[A-Z][a-z]/g, letter => ` ${letter.toLowerCase()}`)
+        .replace(/_/g, " ")
+        .trim()
+        .toLowerCase()
+        .replace(/(^\w|\s\w)/g, m => m.toUpperCase())
```

Comment:
since we have lodash in our package.json already, we could do `_.flow([_.lowerCase, _.capitalize])(this.name)`

Reply:
I prefer to minimise lodash usage, and I hope to remove it one day :)